### PR TITLE
20535 issues with weather demo

### PR DESCRIPTION
**Pull request by jomar-honsi**
- Fixed inverted logic bug
- More readable date format
- Removed some executable code from configuration

### CODE REVIEW

File: samples/dashboards/demo/live-weather/demo.js [hunk 1]
```diff
@@ -584,6 +591,17 @@ async function setupDashboard() {
...
+                            tooltip: {
+                                pointFormatter: function () {
+                                    return this.y + ' ' + activeParam.unit;
+                                }
+                            }
+                        },
+                        windbarb: {
+                            tooltip: {
+                                pointFormat: '<b>{point.value} m/s</b> ({point.beaufort})<br/>Wind direction: {point.direction} degrees',
+                                pointFormatter: null
```
Comment (TorsteinHonsi) [inline samples/dashboards/demo/live-weather/demo.js:604]: `pointFormat` should take precedence, so no need to set `pointFormatter` to null.

**Review by karolkolodziej — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]
Looks good!

File: samples/dashboards/demo/live-weather/demo.js (hunk 1 above)
Comment (jomar-honsi) [reply samples/dashboards/demo/live-weather/demo.js:604]: When not setting pointFormat to null, this is what appears:
image

Comment (TorsteinHonsi) [reply samples/dashboards/demo/live-weather/demo.js:604]: Okay 👍

**Review by Vazonik — APPROVED** [review]

**Review by stitot — COMMENTED** [review]
I mentioned this in my original review, but leave it for now if not possible/easy:

I believe the GUI/visuals would improve if the elevation KPI was a header spanning the three clickable elements. See attached rough sketch. I assume it would need to be an HTML component, and not sure how easy it is to programmatically change the content of those.

If this is to timeconsuming I believe the original version with KPIs in a two by two grid is better, to get rid of that whitespace below.

Other than that: beautiful!

Skjermbilde 2024-01-30 kl  11 30 47

**Comment by jomar-honsi** [comment]
> I mentioned this in my original review, but leave it for now if not possible/easy:
> 
> I believe the GUI/visuals would improve if the elevation KPI was a header spanning the three clickable elements. See attached rough sketch. I assume it would need to be an HTML component, and not sure how easy it is to programmatically change the content of those.
> 
> If this is to timeconsuming I believe the original version with KPIs in a two by two grid is better, to get rid of that whitespace below.
> 
> Other than that: beautiful!
> 
> Skjermbilde 2024-01-30 kl 11 30 47

The proposal looks good and I will give it a try. There will probably be more work in reshuffling the layout than in changes to the code.

**Pull request merged**